awslabs/eventbridge-kafka-connector
# support custom AWS credential provider

Description
-----------

Enable custom credential provider support in the connector so that customers can implement their own provider that will be able support their specific security need.

Test Steps
-----------

```bash
$ mvn clean verify -Drevision=$(git describe --tags --always)
```

Checklist:
----------




Related Issue
-----------

#366


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
